event-svc/src/event/service.rs

```diff
@@ -11,6 +11,7 @@ use super::{
 };
 use async_trait::async_trait;
 use ceramic_core::{EventId, Network, NodeId, SerializeExt};
+use ceramic_event::unvalidated::data::SHOULD_INDEX_DEFAULT;
```

Comment:
How does this default value affect indexing behavior?
Are there any differences in processing between an explicit false, an absent field, and a None value?